components/04-views/_components.scss

```diff
@@ -10,6 +10,7 @@
 @import 'catalog/grid/grid';
 @import 'catalog/list/list';
 @import 'dashboard/default/dashboard';
+@import 'forgot-password/forgot-password';
```

Comment:
Code is ok, I just wonder if you'll use captcha in other view b/c maybe will be better to create separate component for captcha. But is not a bug is only suggestion. If you won't need it in the future, there is no need to do it.

Reply:
I'm not sure if it'll be used later, I'll prefer to leave it here for now. It can be separated later if needed

Reply:
👍